styles/palette.css

```diff
@@ -36,6 +36,9 @@
   --stanford-sky-dark-rgb: 1, 104, 149;
   --stanford-sky-light: rgb(var(--stanford-sky-light-rgb));
   --stanford-sky-dark: rgb(var(--stanford-sky-dark-rgb));
+  --stanford-plum-rgb: 60, 100, 0, 22;
```

Comment:
This is my mistake, I accidentally copied the CMYK values instead of the RGB values in the issue. The RGB values are rgb(98, 0, 89).